doc/2/api/controllers/auth/check-token/index.md

```diff
@@ -40,6 +40,9 @@ Body:
 
 ---
 
+<SinceBadge version="2.16.8">
+When no token is provided the method returns information about the anonymous token, instead of throwing an error.
```

Comment:
```suggestion
When no token is provided the method returns information about the anonymous token (`kuid` is `-1`), instead of throwing an error.
```